SE-XXX Typed throws.md

```diff
@@ -490,16 +537,43 @@ func fooThrower() throws Foo {
     guard someOtherCondition else {
         throw .baz
     }
-
-    [...]
 }
-    
+```
+
+2. In a similar way, a function that `throws` an `enum` type, can be catched with each of its cases individually declared, in different `catch` clauses that can omit the type of which they belong to, as it is inferred from the function signature being called in the `do` block.

```

Comment:
I would change that to

Assuming it is the only thrown error type in the `do` block, an `enum` based error thrown by a typed `throws` can have its cases catched, with each case having a separate catch clause. When catching cases the type of the case can be omitted, as it is inferred from the throwing function.